python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -232,7 +232,11 @@ class Job(GangaObject):
 
     # TODO: usage of **kwds may be envisaged at this level to optimize the
     # overriding of values, this must be reviewed
-    def __init__(self):
+    def __init__(self, old_job=None):
```

Comment:
I think I'd prefer something like `job_tpl` or maybe`prev_job` as `old_job` could be misleading. Also, some docstrings + comments just to document what's happening?